corporate/models.py

```diff
@@ -148,10 +159,19 @@ class LicenseLedger(models.Model):
     """
 
     plan: CustomerPlan = models.ForeignKey(CustomerPlan, on_delete=CASCADE)
+
     # Also True for the initial upgrade.
     is_renewal: bool = models.BooleanField(default=False)
+
     event_time: datetime.datetime = models.DateTimeField()
+
+    # The number of licenses ("seats") purchased by the the organization at the time of ledger entry creation. Normally, to add a user
+    # the organization needs at least one spare license. Once a license is purchased, it is valid till the end of the billing period,
+    # irrespective of whether the license is used or not. So the value of licenses will never decrease for subsequent LicenseLedger entries
+    # in the same billing period.
     licenses: int = models.IntegerField()
-    # None means the plan does not automatically renew.
-    # This cannot be None if plan.automanage_licenses.
+
+    # The number of licenses the organization needs in the next billing cycle. The value of licenses_at_next_renewal can increase or decrease
+    # for subsequent LicenseLedger entries in the same billing period. For plans on automatic license management this value is usually equal to
+    # the number of activated users in the organization.
```

Comment:
Not a huge deal but do we have any rules on wrapping comments at a certain line length? Wrapping at 80 makes it super easy to read files side by side.

Reply:
Our codebase as a whole automatically wraps at 100 for code; it can't easily do that for comments because one might have a 200-character link in it.  I think this is well beyond that, though.

Reply:
Wrapped at 100.